airgun/entities/virtwho_configure.py

```diff
@@ -0,0 +1,121 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.virtwho_configure import (
+    VirtwhoConfigureCreateView,
+    VirtwhoConfigureEditView,
+    VirtwhoConfiguresView,
+    VirtwhoConfigureDetailsView,
+)
+
+
+class VirtwhoConfigureEntity(BaseEntity):
+
+    def _reset_values(self, values):
+        mapping = {
+            'esx': 'VMware vSphere / vCenter (esx)',
+            'xen': 'XenServer (xen)',
+            'hyperv': 'Microsoft Hyper-V (hyperv)',
+            'rhevm': 'Red Hat Virtualization Hypervisor (rhevm)',
+            'libvirt': 'libvirt',
+            'kubevirt': 'Container-native virtualization',
+        }
+        vals = values.copy()
+        if 'hypervisor_type' in vals:
+            hypervisor_value = vals['hypervisor_type']
+            vals.update({'hypervisor_type': mapping[hypervisor_value]})
+        return vals
+
+    def create(self, values):
+        """Create new virtwho configure entity"""
+        view = self.navigate_to(self, 'New')
+        values = self._reset_values(values)
```

Comment:
I would think we should not do such resetting in airgun code rather good to get it from the test itself (test data). Did you refer any code from airgun does such on fly change?

Reply:
Basically, this translates enum to string in UI. Since Airgun is our library for UI, I strongly feel Airgun is exactly the right place to do it. Tests should not be concerned with UI strings - we don't want to update all tests when string changes.

Yes, we already do conceptually same thing in Airgun, see https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/contenthost.py#L120 .

Please also see discussion in https://github.com/SatelliteQE/robottelo/pull/7102#discussion_r299848528

Reply:
@mirzal , what's your opinion? should we move the reset_values back to the test file (`tests/foreman/virtwho/test_ui_virtwho.py`)

Reply:
Ok sure

Reply:
@eko999 No, I think it should be in Airgun.